docs/concepts.rst

```diff
@@ -33,6 +33,9 @@ In Airflow, a ``DAG`` -- or a Directed Acyclic Graph -- is a collection of all
 the tasks you want to run, organized in a way that reflects their relationships
 and dependencies.
 
+A DAG is defined in a Python script, which represents the DAGs structure (tasks
+and their dependencies) as code.
```

Comment:
While this is mentioned later on, it is a concept that is often misunderstood (even the tutorial had to emphasise this). I still have colleagues who don't realize a DAG is actually code...